### DISCUSSION TITLE
Update project to symfony 6

### DISCUSSION THREAD

**Discussion by dev-pigma** [SolidInvoice/SolidInvoice#462] [General]
Hello, I am going to update the project to symfony 6, can I hope for help for this?

**Comment by pierredup** [comment]
Hey @dev-pigma, upgrading Symfony would definitely be welcome and we can give full support for it! However, since we are only on Symfony 4.4 at the moment, the current plan is to get to al least Symfony 5.4 first, before moving to Symfony 6. Moving to Symfony 6 also means dropping support for PHP < 8.1, which I think we only want to do after a couple more releases.

**Comment by pierredup** [comment]
The upgrade to Symfony 5.4 has been done in #482, which will be released as part of version 2.2. After that release, then we can start looking into an upgrade to Symfony 6
